src/screens/Ticketing/Tickets/Tabs.tsx

```diff
@@ -23,18 +23,14 @@ type Props = {
 export const BuyTickets: React.FC<Props> = ({navigation}) => {
   const styles = useStyles();
   const {theme} = useTheme();
-  const {must_upgrade_ticketing, enable_recent_tickets} = useRemoteConfig();
+  const {must_upgrade_ticketing} = useRemoteConfig();
   const {t} = useTranslation();
 
   if (must_upgrade_ticketing) return <UpgradeSplash />;
 
   return (
     <View style={styles.container}>
-      {enable_recent_tickets ? (
-        <RecentTicketsScrollView />
-      ) : (
-        <View style={{flex: 1}} />
-      )}
+      <RecentTicketsScrollView />
```

Comment:
No longer need for feature toggle or unintended change?

Reply:
Both! I'll move it to its own PR.